regolith/builders/publistbuilder.py

```diff
@@ -76,6 +76,26 @@ def latex(self):
                 cat_grants = cat_grants + "_" + g
             filestub = filestub + "{}".format(cat_grants)
             qualifiers = qualifiers + " from Grant{} {}".format(pl, text_grants)
+        if self.rc.kwargs:
+            kw = True
+            kwargs = self.rc.kwargs
+            if isinstance(kwargs, str):
+                key, value = kwargs.split(':', 1)  # Split only at the first colon
+                if key == 'facilities':
+                    facilities = value
+                    filestub = filestub + "_facility_{}".format(facilities)
+                    qualifiers = qualifiers + " from facility {}".format(facilities)
+                elif key == 'doi':
```

Comment:
i don't think we want this, so remove it.  I guess you need it to make the tests pass, but it is polluting the code.  It presumably means we have to update the schema :(

I will merge this so I can use the code, but let's make an issue and fix this right away.

Reply:
For sure. Just to make sure I understand, you mean about the key == doi part?